<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="windowModality">
   <enum>Qt::ApplicationModal</enum>
  </property>
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1010</width>
    <height>600</height>
   </rect>
  </property>
  <property name="contextMenuPolicy">
   <enum>Qt::NoContextMenu</enum>
  </property>
  <property name="windowTitle">
   <string/>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(85, 87, 83)</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QWidget" name="horizontalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>70</y>
      <width>1001</width>
      <height>521</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <item>
        <widget class="QWidget" name="widget" native="true">
         <widget class="QPushButton" name="pb_battery">
          <property name="geometry">
           <rect>
            <x>0</x>
            <y>-6</y>
            <width>331</width>
            <height>521</height>
           </rect>
          </property>
          <property name="styleSheet">
           <string notr="true">background-color: rgb(14, 86, 22)</string>
          </property>
          <property name="text">
           <string/>
          </property>
         </widget>
         <widget class="QLabel" name="l_battery">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="geometry">
           <rect>
            <x>0</x>
            <y>0</y>
            <width>331</width>
            <height>41</height>
           </rect>
          </property>
          <property name="toolTip">
           <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;br/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
          </property>
          <property name="whatsThis">
           <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;br/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
          </property>
          <property name="styleSheet">
           <string notr="true">background-color: rgba(16, 73, 31,40);
font: 20pt &quot;Ubuntu&quot;;</string>
          </property>
          <property name="text">
           <string>  Battery SOC</string>
          </property>
         </widget>
         <widget class="QLabel" name="l_SOC_value">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="geometry">
           <rect>
            <x>10</x>
            <y>50</y>
            <width>161</width>
            <height>91</height>
           </rect>
          </property>
          <property name="styleSheet">
           <string notr="true">background-color: rgba(16, 73, 31,0);
font: 50pt &quot;Ubuntu&quot;;</string>
          </property>
          <property name="text">
           <string> 100</string>
          </property>
         </widget>
         <widget class="QLabel" name="label_3">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="geometry">
           <rect>
            <x>220</x>
            <y>50</y>
            <width>81</width>
            <height>91</height>
           </rect>
          </property>
          <property name="styleSheet">
           <string notr="true">background-color: rgba(16, 73, 31,0);
font: 50pt &quot;Ubuntu&quot;;</string>
          </property>
          <property name="text">
           <string>%</string>
          </property>
         </widget>
         <widget class="QLabel" name="l_battery_15">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="geometry">
           <rect>
            <x>0</x>
            <y>160</y>
            <width>331</width>
            <height>41</height>
           </rect>
          </property>
          <property name="toolTip">
           <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;br/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
          </property>
          <property name="whatsThis">
           <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;br/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
          </property>
          <property name="styleSheet">
           <string notr="true">background-color: rgba(16, 73, 31,40);
font: 20pt &quot;Ubuntu&quot;;</string>
          </property>
          <property name="text">
           <string>  Power Temperature</string>
          </property>
         </widget>
         <widget class="QLabel" name="l_powertemp_value">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="geometry">
           <rect>
            <x>20</x>
            <y>210</y>
            <width>161</width>
            <height>91</height>
           </rect>
          </property>
          <property name="styleSheet">
           <string notr="true">background-color: rgba(16, 73, 31,0);
font: 50pt &quot;Ubuntu&quot;;</string>
          </property>
          <property name="text">
           <string>  44</string>
          </property>
         </widget>
         <widget class="QLabel" name="label_5">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="geometry">
           <rect>
            <x>220</x>
            <y>210</y>
            <width>81</width>
            <height>91</height>
           </rect>
          </property>
          <property name="styleSheet">
           <string notr="true">background-color: rgba(16, 73, 31,0);
font: 50pt &quot;Ubuntu&quot;;</string>
          </property>
          <property name="text">
           <string>C°</string>
          </property>
         </widget>
         <widget class="QLabel" name="label_6">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="geometry">
           <rect>
            <x>190</x>
            <y>390</y>
            <width>111</width>
            <height>91</height>
           </rect>
          </property>
          <property name="styleSheet">
           <string notr="true">background-color: rgba(16, 73, 31,0);
font: 50pt &quot;Ubuntu&quot;;</string>
          </property>
          <property name="text">
           <string>KW</string>
          </property>
         </widget>
         <widget class="QLabel" name="l_power_value">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="geometry">
           <rect>
            <x>20</x>
            <y>390</y>
            <width>161</width>
            <height>91</height>
           </rect>
          </property>
          <property name="styleSheet">
           <string notr="true">background-color: rgba(16, 73, 31,0);
font: 50pt &quot;Ubuntu&quot;;</string>
          </property>
          <property name="text">
           <string>  10</string>
          </property>
         </widget>
         <widget class="QLabel" name="l_battery_16">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="geometry">
           <rect>
            <x>0</x>
            <y>340</y>
            <width>331</width>
            <height>41</height>
           </rect>
          </property>
          <property name="toolTip">
           <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;br/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
          </property>
          <property name="whatsThis">
           <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;br/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
          </property>
          <property name="styleSheet">
           <string notr="true">background-color: rgba(16, 73, 31,40);
font: 20pt &quot;Ubuntu&quot;;</string>
          </property>
          <property name="text">
           <string>  Motor Power</string>
          </property>
         </widget>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <widget class="QWidget" name="widget_2" native="true">
         <widget class="QPushButton" name="pb_motor">
          <property name="geometry">
           <rect>
            <x>0</x>
            <y>-6</y>
            <width>331</width>
            <height>521</height>
           </rect>
          </property>
          <property name="styleSheet">
           <string notr="true">background-color: rgb(9, 80, 99);
</string>
          </property>
          <property name="text">
           <string/>
          </property>
         </widget>
         <widget class="QLabel" name="label_7">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="geometry">
           <rect>
            <x>220</x>
            <y>210</y>
            <width>81</width>
            <height>91</height>
           </rect>
          </property>
          <property name="styleSheet">
           <string notr="true">background-color: rgba(16, 73, 31,0);
font: 50pt &quot;Ubuntu&quot;;</string>
          </property>
          <property name="text">
           <string>C°</string>
          </property>
         </widget>
         <widget class="QLabel" name="l_speed_value">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="geometry">
           <rect>
            <x>10</x>
            <y>50</y>
            <width>161</width>
            <height>91</height>
           </rect>
          </property>
          <property name="styleSheet">
           <string notr="true">background-color: rgba(16, 73, 31,0);
font: 50pt &quot;Ubuntu&quot;;</string>
          </property>
          <property name="text">
           <string>  00</string>
          </property>
         </widget>
         <widget class="QLabel" name="label_9">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="geometry">
           <rect>
            <x>190</x>
            <y>50</y>
            <width>111</width>
            <height>91</height>
           </rect>
          </property>
          <property name="styleSheet">
           <string notr="true">background-color: rgba(16, 73, 31,0);
font: 30pt &quot;Ubuntu&quot;;</string>
          </property>
          <property name="text">
           <string>Km/h</string>
          </property>
         </widget>
         <widget class="QLabel" name="label_10">
          <property name="enabled">
           <bool>true</bool>
          </property>
          <property name="geometry">
           <rect>
            <x>190</x>
            <y>390</y>
            <width>111</width>
            <height>91</height>
           </rect>
          </property>
          <property name="styleSheet">
           <string notr="true">background-color: rgba(16, 73, 31,0);
font: 50pt &quot;Ubuntu&quot;;</string>
          </property>
          <property name="text">
           <string/>
          </property>
         </widget>
         <widget class="QLabel" name="l_motortemp_value">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="geometry">
           <rect>
            <x>20</x>
            <y>210</y>
            <width>161</width>
            <height>91</height>
           </rect>
          </property>
          <property name="styleSheet">
           <string notr="true">background-color: rgba(16, 73, 31,0);
font: 50pt &quot;Ubuntu&quot;;</string>
          </property>
          <property name="text">
           <string>  20</string>
          </property>
         </widget>
         <widget class="QLabel" name="l_battery_2">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="geometry">
           <rect>
            <x>0</x>
            <y>0</y>
            <width>331</width>
            <height>41</height>
           </rect>
          </property>
          <property name="toolTip">
           <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;br/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
          </property>
          <property name="whatsThis">
           <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;br/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
          </property>
          <property name="styleSheet">
           <string notr="true">background-color: rgba(16, 73, 31,40);
font: 20pt &quot;Ubuntu&quot;;</string>
          </property>
          <property name="text">
           <string>  Speed</string>
          </property>
         </widget>
         <widget class="QLabel" name="l_battery_17">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="geometry">
           <rect>
            <x>0</x>
            <y>160</y>
            <width>331</width>
            <height>41</height>
           </rect>
          </property>
          <property name="toolTip">
           <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;br/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
          </property>
          <property name="whatsThis">
           <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;br/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
          </property>
          <property name="styleSheet">
           <string notr="true">background-color: rgba(16, 73, 31,40);
font: 20pt &quot;Ubuntu&quot;;</string>
          </property>
          <property name="text">
           <string>  Motor Temperature</string>
          </property>
         </widget>
         <widget class="QLabel" name="l_RPM_value">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="geometry">
           <rect>
            <x>20</x>
            <y>390</y>
            <width>161</width>
            <height>91</height>
           </rect>
          </property>
          <property name="styleSheet">
           <string notr="true">background-color: rgba(16, 73, 31,0);
font: 50pt &quot;Ubuntu&quot;;</string>
          </property>
          <property name="text">
           <string>  40</string>
          </property>
         </widget>
         <widget class="QLabel" name="l_battery_18">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="geometry">
           <rect>
            <x>0</x>
            <y>340</y>
            <width>331</width>
            <height>41</height>
           </rect>
          </property>
          <property name="toolTip">
           <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;br/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
          </property>
          <property name="whatsThis">
           <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;br/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
          </property>
          <property name="styleSheet">
           <string notr="true">background-color: rgba(16, 73, 31,40);
font: 20pt &quot;Ubuntu&quot;;</string>
          </property>
          <property name="text">
           <string>  RPM</string>
          </property>
         </widget>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QWidget" name="widget_3" native="true">
         <widget class="QPushButton" name="pb_temperature">
          <property name="geometry">
           <rect>
            <x>0</x>
            <y>-6</y>
            <width>331</width>
            <height>291</height>
           </rect>
          </property>
          <property name="styleSheet">
           <string notr="true">background-color:rgb(60, 76, 84)</string>
          </property>
          <property name="text">
           <string/>
          </property>
         </widget>
         <widget class="QLabel" name="l_temperature">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="geometry">
           <rect>
            <x>0</x>
            <y>0</y>
            <width>331</width>
            <height>41</height>
           </rect>
          </property>
          <property name="toolTip">
           <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;br/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
          </property>
          <property name="whatsThis">
           <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;br/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
          </property>
          <property name="styleSheet">
           <string notr="true">background-color: rgba(16, 73, 31,40);
font: 20pt &quot;Ubuntu&quot;;</string>
          </property>
          <property name="text">
           <string>   Temperature</string>
          </property>
         </widget>
         <widget class="QLabel" name="label_20">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="geometry">
           <rect>
            <x>180</x>
            <y>170</y>
            <width>91</width>
            <height>31</height>
           </rect>
          </property>
          <property name="styleSheet">
           <string notr="true">background-color: rgba(16, 73, 31,0);
font: 15pt &quot;Ubuntu&quot;;</string>
          </property>
          <property name="text">
           <string>Voltage:</string>
          </property>
         </widget>
         <widget class="QLabel" name="label_21">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="geometry">
           <rect>
            <x>40</x>
            <y>70</y>
            <width>121</width>
            <height>51</height>
           </rect>
          </property>
          <property name="styleSheet">
           <string notr="true">background-color: rgba(16, 73, 31,0);
font: 40pt &quot;Ubuntu&quot;;</string>
          </property>
          <property name="text">
           <string>  33</string>
          </property>
         </widget>
         <widget class="QLabel" name="label_22">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="geometry">
           <rect>
            <x>180</x>
            <y>200</y>
            <width>81</width>
            <height>61</height>
           </rect>
          </property>
          <property name="styleSheet">
           <string notr="true">background-color: rgba(16, 73, 31,0);
font: 30pt &quot;Ubuntu&quot;;</string>
          </property>
          <property name="text">
           <string>200</string>
          </property>
         </widget>
         <widget class="QLabel" name="label_23">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="geometry">
           <rect>
            <x>180</x>
            <y>70</y>
            <width>81</width>
            <height>51</height>
           </rect>
          </property>
          <property name="styleSheet">
           <string notr="true">background-color: rgba(16, 73, 31,0);
font: 40pt &quot;Ubuntu&quot;;</string>
          </property>
          <property name="text">
           <string/>
          </property>
         </widget>
         <widget class="QLabel" name="label_24">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="geometry">
           <rect>
            <x>10</x>
            <y>200</y>
            <width>81</width>
            <height>61</height>
           </rect>
          </property>
          <property name="styleSheet">
           <string notr="true">background-color: rgba(16, 73, 31,0);
font: 30pt &quot;Ubuntu&quot;;</string>
          </property>
          <property name="text">
           <string>  33</string>
          </property>
         </widget>
         <widget class="QLabel" name="label_25">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="geometry">
           <rect>
            <x>90</x>
            <y>200</y>
            <width>41</width>
            <height>61</height>
           </rect>
          </property>
          <property name="styleSheet">
           <string notr="true">background-color: rgba(16, 73, 31,0);
font: 30pt &quot;Ubuntu&quot;;</string>
          </property>
          <property name="text">
           <string>A</string>
          </property>
         </widget>
         <widget class="QLabel" name="label_26">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="geometry">
           <rect>
            <x>20</x>
            <y>40</y>
            <width>71</width>
            <height>31</height>
           </rect>
          </property>
          <property name="styleSheet">
           <string notr="true">background-color: rgba(16, 73, 31,0);
font: 15pt &quot;Ubuntu&quot;;</string>
          </property>
          <property name="text">
           <string>RPM:</string>
          </property>
         </widget>
         <widget class="QLabel" name="l_battery_6">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="geometry">
           <rect>
            <x>0</x>
            <y>150</y>
            <width>331</width>
            <height>16</height>
           </rect>
          </property>
          <property name="toolTip">
           <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;br/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
          </property>
          <property name="whatsThis">
           <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;br/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
          </property>
          <property name="styleSheet">
           <string notr="true">background-color: rgba(16, 73, 31,40);
font: 20pt &quot;Ubuntu&quot;;</string>
          </property>
          <property name="text">
           <string/>
          </property>
         </widget>
         <widget class="QLabel" name="label_27">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="geometry">
           <rect>
            <x>260</x>
            <y>200</y>
            <width>41</width>
            <height>61</height>
           </rect>
          </property>
          <property name="styleSheet">
           <string notr="true">background-color: rgba(16, 73, 31,0);
font: 30pt &quot;Ubuntu&quot;;</string>
          </property>
          <property name="text">
           <string>V</string>
          </property>
         </widget>
         <widget class="QLabel" name="label_28">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="geometry">
           <rect>
            <x>10</x>
            <y>170</y>
            <width>81</width>
            <height>31</height>
           </rect>
          </property>
          <property name="styleSheet">
           <string notr="true">background-color: rgba(16, 73, 31,0);
font: 15pt &quot;Ubuntu&quot;;</string>
          </property>
          <property name="text">
           <string>Current:</string>
          </property>
         </widget>
         <widget class="QLabel" name="l_battery_7">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="geometry">
           <rect>
            <x>150</x>
            <y>150</y>
            <width>20</width>
            <height>131</height>
           </rect>
          </property>
          <property name="toolTip">
           <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;br/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
          </property>
          <property name="whatsThis">
           <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;br/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
          </property>
          <property name="styleSheet">
           <string notr="true">background-color: rgba(16, 73, 31,40);
font: 20pt &quot;Ubuntu&quot;;</string>
          </property>
          <property name="text">
           <string/>
          </property>
         </widget>
         <widget class="QLabel" name="l_logo">
          <property name="geometry">
           <rect>
            <x>-30</x>
            <y>290</y>
            <width>371</width>
            <height>291</height>
           </rect>
          </property>
          <property name="styleSheet">
           <string notr="true">image: url(:/images/logotrasparent.png) ;

</string>
          </property>
          <property name="text">
           <string/>
          </property>
         </widget>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
   <widget class="QLabel" name="l_date">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>411</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgba(75, 77, 73, 40);
font: 15pt &quot;Ubuntu&quot;;</string>
    </property>
    <property name="text">
     <string>  14:30 - Friday, 27 January 2023</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pb_faults">
    <property name="geometry">
     <rect>
      <x>930</x>
      <y>10</y>
      <width>61</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(164, 0, 0)</string>
    </property>
    <property name="text">
     <string>12</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pb_boatinfo">
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>10</y>
      <width>181</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(32, 74, 135)</string>
    </property>
    <property name="text">
     <string>Boat info</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pb_settings">
    <property name="geometry">
     <rect>
      <x>590</x>
      <y>10</y>
      <width>191</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>Settings</string>
    </property>
   </widget>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
